src/index.ts

```diff
@@ -1,2 +1,118 @@
-import { getMessage } from './util';
-console.log(getMessage());
\ No newline at end of file
+import fs from 'node:fs';
+import path from 'node:path';
+import prompts from 'prompts';
+import { red, reset } from 'kolorist';
+import { getTemplateDirPath, copy, mkDir } from './util';
+
+const cwd = process.cwd();
+const dotGithubDir = path.join(cwd, '.github');
+const issueTemplateDir = path.join(dotGithubDir, 'ISSUE_TEMPLATE');
+
+async function init() {
+  let result: prompts.Answers<
+    'prTemplate' | 'contribTemplate' | 'issueTemplate'
+  >;
+
+  try {
+    result = await prompts(
+      [
+        {
+          type: 'confirm',
+          name: 'prTemplate',
+          message: reset('Do you want Pr Template:'),
+        },
+        {
+          type: 'confirm',
+          name: 'contribTemplate',
+          message: reset('Do you want Contrib Template:'),
+        },
+        {
+          type: 'confirm',
+          name: 'issueTemplate',
+          message: reset('Do you want issue Template:'),
+        },
+      ],
+      {
+        onCancel: () => {
+          throw new Error(red('✖') + ' Operation cancelled');
+        },
+      },
+    );
+  } catch (cancelled: any) {
+    console.log(cancelled.message);
+    return;
+  }
+
+  // user choice associated with prompts
+  const { prTemplate, contribTemplate, issueTemplate } = result;
+  if (!prTemplate && !contribTemplate && !issueTemplate) {
+    console.log(`\nExiting...`);
+    return;
+  }
+
+  const root = path.join(cwd);
+  console.log(`\nScaffolding templates in ${root}...`);
+
+  if (contribTemplate) {
+    const templateDir = getTemplateDirPath('contrib');
+    const files = fs.readdirSync(templateDir);
```

Comment:
@nabeel-shakeel `fs.readdirSync` will never return undefined, because the templates folder, where we are reading templates from will be part of our package bundle in our npm package.